misc/entitlements/debug.entitlements

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Might `xcode_entitlements` or `macos` be a better directory name?
That way it would be visible from the directory name (or path) what it is used for, without opening and reading the files or searching for references in CMakeLists.txt.

`Info.plist` could be another candidate to move here so that MacOS-specific files are located in the same location. `util/xcode` already makes a proper distinction, which is fine as-is.

Reply:
I did not do sub dir because `Info.plist` is flat as another OS depends on files.

But not a problem to move to `macos` sub dir. Should I also move `minetest-icon.icns` (it is being used only for macOS build)?

Reply:
Thanks for clarifying.
IMO it would make sense to move the `.icns` file. But it's not introduced by this PR, thus I'll not insist on it and leave the decision up to you.

Reply:
moved